### DISCUSSION TITLE
Links and question

### DISCUSSION THREAD

**Discussion by PVerburg** [jensrossbach/node-red-sony-audio#10] [Feedback]
Hi, thanks for your work on this I'm trying to understand your flows I have been trying to control my Sony STR-AN1000 with CEC and not much luck. 
This looks very promising and being retired is making learning new things a bit harder, first my Sony amp has an AUX but your status says line no problem I can work with that.
Also the links to the sony API docs are broken they might help me understanding the proccess I was trying to debug the output to the unit i.e. the actual full call but that seems to be unavailable ???
Again thanks keep up the good work.
Paul V.

**Comment by jensrossbach** [comment]
Hi @PVerburg!

> This looks very promising and being retired is making learning new things a bit harder, first my Sony amp has an AUX but your status says line no problem I can work with that.

I'm happy that you like the nodes. I used the term "line" as it is named like this in the Audio Control API.

> Also the links to the sony API docs are broken they might help me understanding the proccess I was trying to debug the output to the unit i.e. the actual full call but that seems to be unavailable ???

Yes, unfortunately. I also noticed this already but this seems to be on Sony side. If you search for Sony Audio Control API in Google or Bing, you also get these broken links listed, so I assume, Sony just removed the documentation. The Sony Audio Control API has been marked as deprecated since one or two years and maybe this is the reason why they removed the docs now. Some weeks ago the link was already broken but it was still listed on the Sony developer website under the archive. But now this archive is also removed, the Sony developer website has been reduced a lot and there is not much information left. Maybe we are lucky and they are only reworking the developer website and will bring the Audio Control API documentation back some day.

**Reply by PVerburg** [reply]
would you have a copy of the docs that you could share just for my reference?
I noticed that my display is now blank until I send a command or change a setting on the amp nice but how did I do it lol.
I have been using CEC and not happy so will try this, I somewhere saw a way of extracting all available commands in the API but don't seem to be able to find it again could you assist ? 
Thanks.

**Reply by jensrossbach** [reply]
Unfortunately I didn't create a copy of the docs as I didn't expect that they will be removed.

The API for retrieving a list of supported APIs is `getSupportedApiInfo`, maybe it can help you.
